.vscode/settings.json

```diff
@@ -1,23 +1,22 @@
 {
-  "python.linting.enabled": true,
-  "python.linting.flake8Enabled": true,
-  "python.linting.mypyEnabled": true,
-  "python.formatting.provider": "black",
+  "black-formatter.importStrategy": "fromEnvironment",
   "pylint.importStrategy": "fromEnvironment",
+  "pylint.args": [
+    "--load-plugins pylint_pydantic",
+    "--rcfile=pylintrc"
+  ],
   "flake8.importStrategy": "fromEnvironment",
-  "black-formatter.importStrategy": "fromEnvironment",
-  "python.linting.flake8Args": [
+  "flake8.args": [
     "--config=/dev/null",
     "--max-line-length=165"
   ],
-  "python.linting.mypyArgs": [
+  "mypy-type-checker.importStrategy": "fromEnvironment",
+  "mypy-type-checker.args": [
     "--config-file=pyproject.toml"
   ],
-  "pylint.severity": {
-    "refactor": "Warning"
+  "isort.importStrategy": "fromEnvironment",
+  "isort.check": true,
+  "files.associations": {
+    "*.yaml": "home-assistant"
```

Comment:
so you are using home-assistant with ANTA ? :p

Reply:
f**k I am fighting with this extension